### PR TITLE
[Python Wheels] Add dockerfiles

### PR DESCRIPTION
**Pull request by HubertBalcerzak**
This moves Wheelbuilder dockerfiles to the Kratos repository

### CODE REVIEW

File: scripts/docker_files/docker_file_wheelbuilder_windows/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    c:\python\38\python.exe -m pip install --upgrade setuptools wheel
+
+#Downloand blas/lapack
+RUN powershell.exe -Command \
+	[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
+	mkdir c:\libs; \
+	wget https://icl.cs.utk.edu/lapack-for-windows/libraries/VisualStudio/3.7.0/Dynamic-MINGW/Win64/libblas.lib -OutFile c:\libs\libblas.lib; \
+	wget https://icl.cs.utk.edu/lapack-for-windows/libraries/VisualStudio/3.7.0/Dynamic-MINGW/Win64/liblapack.lib -OutFile c:\libs\liblapack.lib
+
+#Download eigenlib
```
Comment (philbucher) [inline scripts/docker_files/docker_file_wheelbuilder_windows/Dockerfile]: This is contained in the App itself meanwhile, you can remove it here

**Review by philbucher — COMMENTED** [review]
Awesome!
I noticed one thing while browsing through

File: scripts/docker_files/docker_file_wheelbuilder_windows/Dockerfile (hunk 1 above)
Comment (HubertBalcerzak) [reply scripts/docker_files/docker_file_wheelbuilder_windows/Dockerfile]: oh, that's nice

Comment (philbucher) [reply scripts/docker_files/docker_file_wheelbuilder_windows/Dockerfile]: I meant the Eigen lib, Blas and Lapack are still needed (only for the ExternalSolversApplication)

Comment (HubertBalcerzak) [reply scripts/docker_files/docker_file_wheelbuilder_windows/Dockerfile]: In that case I'm lost, why would I want to remove them?

Comment (philbucher) [reply scripts/docker_files/docker_file_wheelbuilder_windows/Dockerfile]: To me it seems like you removed Blas & Lapack in your last commit: https://github.com/KratosMultiphysics/Kratos/pull/6633/commits/f4a6f16dba4d5b572cc0cb5c20b8d0261d40c44b

Comment (HubertBalcerzak) [reply scripts/docker_files/docker_file_wheelbuilder_windows/Dockerfile]: Which was a mistake, since they are still needed by ExternalSolvers

Comment (roigcarlo) [reply scripts/docker_files/docker_file_wheelbuilder_windows/Dockerfile]: @philbucher  Actually if we go ahead with the Eigensolvers app as replacement for the External we could remove those once SuperLu is replaced everywhere. Maybe we can speak it this afternoon, it's still not clear to me if there is still some need for it

File: scripts/docker_files/docker_file_wheelbuilder_windows/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+RUN powershell.exe -Command \
+	[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
+	mkdir c:\eigen; \
+	wget https://gitlab.com/libeigen/eigen/-/archive/3.3.7/eigen-3.3.7.zip -OutFile c:\temp\eigen.zip; \
+	c:\7zip\7z.exe x c:\temp\eigen.zip -o"c:\eigen"
```
Comment (philbucher) [inline scripts/docker_files/docker_file_wheelbuilder_windows/Dockerfile]:
```suggestion

```

thats what I meant :)

@roigcarlo sure lets discuss it this afternoon

File: scripts/docker_files/docker_file_wheelbuilder_windows/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+#set env variables
+ENV CMAKE c:\\cmake\\cmake-3.14.1-win64-x64\\bin\\cmake.exe
+ENV SEVEN_ZIP c:\\7zip\\7z.exe
+ENV VCVARS  \"C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools\\VC\\Auxiliary\\Build\\vcvars64.bat\"
+ENV BOOST c:\\boost\\boost_1_69_0
+ENV BLAS c:\\libs\\libblas.lib
+ENV LAPACK c:\\libs\\liblapack.lib
+ENV PYTHONROOT c:\\python
+ENV EIGEN c:\\eigen\\eigen-3.3.7
```
Comment (philbucher) [inline scripts/docker_files/docker_file_wheelbuilder_windows/Dockerfile]:
```suggestion

```

File: scripts/docker_files/docker_file_wheelbuilder_linux/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+RUN yum install -y libgfortran.x86_64 blas-devel lapack-devel
+
+RUN mkdir -p /workspace/scripts; \
+	mkdir -p /workspace/kratos
+
+RUN mkdir /workspace/boost; \
+	cd /workspace/boost; \
+	wget https://dl.bintray.com/boostorg/release/1.71.0/source/boost_1_71_0.tar.gz; \
+	tar xzf boost_1_71_0.tar.gz; \
+	rm boost_1_71_0.tar.gz
```
Comment (philbucher) [inline scripts/docker_files/docker_file_wheelbuilder_linux/Dockerfile]: is there a reason you use different versions of boost in linux and win?

**Comment by philbucher** [comment]
just for the record, in a future PR I would add an Action that builds the wheels when we create a release

**Comment by philbucher** [comment]
no further comment, nice work :+1:

**Comment by philbucher** [comment]
codacy has some suggestions but I am not sure how applicable they are

File: scripts/docker_files/docker_file_wheelbuilder_windows/Dockerfile (hunk 1 above)
Comment (philbucher) [reply scripts/docker_files/docker_file_wheelbuilder_windows/Dockerfile]: I suggest we clean this up after [this](https://github.com/KratosMultiphysics/Kratos/pull/6482#issuecomment-608456537) is done

**Review by philbucher — APPROVED** [review]
ok for me

**Comment by philbucher** [comment]
@HubertBalcerzak will you create the Action to create and upload the docker-images?

**Comment by HubertBalcerzak** [comment]
@philbucher i'll check if that's even possible with the windows one

**Pull request merged**